### PR TITLE
Create burger menu components

### PR DESCRIPTION
**Pull request by Hamuud**
[DEMO LINK](https://fe-feb23-code-hunters.github.io/fe_phone-catalog/)

### CODE REVIEW

**Comment by asmd564** [comment]
Use module styles (example: style.module.scss). in react (import style from './style.module.scss'

**Review by asmd564 — CHANGES_REQUESTED** [review]
Use module styles (example: style.module.scss). in react (import style from './style.module.scss'

File: src/components/BurgerMenu/BurgerMenu.tsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+import HeartOutlined from '../../icons/HeartOutlined/HeartOutlined';
+import ShoppingBag from '../../icons/ShoppingBag/ShoppingBag';
+
+export const BurgerMenu = () => {
+  return (
+    <nav className="burger" id="menu">
+      <div className="container">
+        <div className="burger__content">
+          <div className="burger__top top-actions">
+            <a href="/" className="logo">
```
Comment (prokivskaaa) [inline src/components/BurgerMenu/BurgerMenu.tsx:12]: Use tag Link

File: src/components/BurgerMenu/BurgerMenu.scss [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+  &__top {
+    height: 48px;
+    padding-left: 16px;
+    margin-bottom: 32px;
+
+    border-bottom: 1px solid $secondary-color;
+
+    &_close {
+      display: block;
+      height: 48px;
```
Comment (prokivskaaa) [inline src/components/BurgerMenu/BurgerMenu.scss:32]: Lets declare local vars for height and padding

File: src/App.tsx [hunk 3]
```diff
@@ -1,8 +1,9 @@
+import { BurgerMenu } from './components/BurgerMenu';
 import './styles/App.scss';
 
 const App = () => (
   <div className="App">
-    <h1>Phone Catalog</h1>
+    <BurgerMenu />
```
Comment (prokivskaaa) [inline src/App.tsx:6]: Deploy one more time as it is not present on GH-pages
Знімок екрана 2023-06-09 о 08 55 49

**Review by prokivskaaa — CHANGES_REQUESTED** [review]
Well done but pay attention to the comments

File: src/index.tsx [hunk 4]
```diff
@@ -1,8 +1,13 @@
...
+import { BrowserRouter } from 'react-router-dom';
 import App from './App';
 
 const root = ReactDOM.createRoot(
   document.getElementById('root') as HTMLElement,
 );
 
-root.render(<App />);
+root.render(
+  <BrowserRouter>
```
Comment (prokivskaaa) [inline src/index.tsx:10]: Use HashRouter instead

File: src/components/BurgerMenu/BurgerMenu.module.scss [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
+
+  text-align: center;
+  list-style: none;
+}
+
+.burger__item {
+  margin-bottom: $burger-item-margin;
+}
+
+.burger__link {
```
Comment (prokivskaaa) [inline src/components/BurgerMenu/BurgerMenu.module.scss:81]: telegram-cloud-photo-size-2-5195406107730889569-x
Yulia learned how to implement bem to the module scss classes, here is an example of how to import them. Maybe we can use it somewhere in order to write less code?

**Review by prokivskaaa — APPROVED** [review]
Brilliant!

**Review by asmd564 — APPROVED** [review]

**Comment by prokivskaaa** [comment]
Can you redeploy, please

**Pull request merged**